### PR TITLE
ElvaRivle/Feature/GeQERfQ9

### PR DESCRIPTION
**Pull request by ElvaRivle**
ElvaRivle/Feature/GeQERfQ9

--------------------------------------------------------------------
 MAIN
 * Fixed modal look
 * Added additional fields, although not fully functional  
--------------------------------------------------------------------
 OTHER
 * Fixed \<Select\> values for Department and Tech Stack  
 * Fixed currency input width
--------------------------------------------------------------------
 TODO
 * make file upload functional
 * implement currency logic (convert all to BAM?)
 * connect with backend

### CODE REVIEW

**Comment by ElvaRivle** [comment]
image

**Review by becarselman — APPROVED** [review]

File: src/components/_addEmployeeModal/utils/FormField.js [hunk 1]
```diff
@@ -13,7 +15,15 @@ const FormField = ({ item }) => {
         case "multiselect":
             return <MultiSelect item={item} />
         case "button":
-            return <Button item={item} />
+            //Button won't be wrapped in <FormField>
```
Comment (becarselman) [inline src/components/_addEmployeeModal/utils/FormField.js]: Please remove unnecessary comments

**Review by becarselman — APPROVED** [review]

File: src/components/_addEmployeeModal/utils/FormField.js (hunk 1 above)
Comment (becarselman) [reply src/components/_addEmployeeModal/utils/FormField.js]: add new screenshot

Comment (ElvaRivle) [reply src/components/_addEmployeeModal/utils/FormField.js]: new screenshot already added, just updated old comment with new picture

**Pull request merged**